ietf/templates/person/merge.html

```diff
@@ -37,16 +37,27 @@ <h1>Merge Person Records</h1>
         {% if method == 'post' %}
             <a class="btn btn-primary"
                href="{% url 'ietf.person.views.merge' %}?source={{ target.pk }}&amp;target={{ source.pk }}"
-               role="button">
+               role="button"
+               title="Swap source and target">
                 Swap
             </a>
         {% endif %}
-        <button type="submit" class="btn btn-warning">
-            {% if method == 'post' %}
+        {% if method == 'post' %}
+            <button type="submit" class="btn btn-warning" title="Merge Person records">
                 Merge
-            {% else %}
-                Submit
-            {% endif %}
-        </button>
+            </button>
+        {% else %}
+            <button type="submit" class="btn btn-warning" title="Get Person information">
+                Get Info
+            </button>
+        {% endif %}
+        {% if method == 'post' %}
+            <a class="btn btn-primary"
+               href="{% url 'ietf.person.views.send_merge_request' %}?source={{ source.pk }}&amp;target={{ target.pk }}"
```

Comment:
Does the \&amp; entity do the right thing here? I think it should be a literal \& since it's in a URL